ui/app/AppLayouts/Profile/controls/CommunityShowcaseDelegate.qml

```diff
@@ -1,8 +1,11 @@
 import QtQuick 2.15
 
+import utils 1.0
+
 ShowcaseDelegate {
     title: !!showcaseObj && !!showcaseObj.name ? showcaseObj.name : ""
-    secondaryTitle: !!showcaseObj && !!showcaseObj.amISectionAdmin ? qsTr("Admin") : qsTr("Member")
+    secondaryTitle: !!showcaseObj && (showcaseObj.memberRole === Constants.memberRole.owner ||
```

Comment:
This is not yet in `master` with community permissions?

Reply:
Oh now it is 😄 I noticed late yesterday the profile showcase was kinda broken due to this... 🥴 But pushed to a wrong branch 😆